src/Kaba.ts

```diff
@@ -423,7 +423,7 @@ export class Kaba
     /**
      * Builds the specialized webpack config for a legacy / module build
      */
-    private buildWebpackConfig (cliConfig: kaba.CliConfig, isModule: boolean): Partial<webpack.Configuration>
+    private buildWebpackConfigs (cliConfig: kaba.CliConfig, isModule: boolean): Partial<webpack.Configuration>[]
```

Comment:
I would refactor this whole method.

As we now always have separate compilers per entry, we can just expand this to separate compilers per entry + module/legacy.

So the `jsConfig` looks something like this:

```js
const compilerConfigs = [];

for (const entry in this.jsEntries)
{
    compilerConfigs.push(this.buildWepackConfig(entry, this.jsEntries[entry], cliConfig, false);

    if (this.buildModern)
    {
        compilerConfigs.push(this.buildWepackConfig(entry, this.jsEntries[entry], cliConfig, true);
    }
}
```

And then in the webpack runner we can just map over all compiler configs.
Now you only have a single method that generates the complete webpack config, which makes everything a bit easier.
In common you basically only need the `watch` flag, so you can just remove `common` and add the `watch` flag directly to `jsConfig`.

What we should improve as well is that we only lint in the `!isModule` build:

1. we only lint once per entry
2. legacy is always built, so we add it in `!isModule` instead of `isModule`.